internal/gatewayapi/testdata/envoyproxy-accesslog-cel-with-invalid.out.yaml

```diff
@@ -18,9 +18,9 @@ gateways:
     conditions:
     - lastTransitionTime: null
       message: 'Invalid access log backendRefs: invalid CEL expression: )++++'
-      reason: Invalid
+      reason: InvalidParameters
       status: "False"
-      type: ListenersNotValid
```

Comment:
this seems not right, GatewayConditionType only allow `Accepted` and `Programmed`.